### DISCUSSION TITLE
React Router and React Redux

### DISCUSSION THREAD

**Discussion by fernandabeatriz1** [community/community#69159] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

What is the difference between React Router and React Redux?

**Comment by ghost** [comment]
React Router and React Redux are two widely used libraries in building applications with React, but they serve different purposes:

React Router:

It's a library for managing routing in React applications.
It helps define multiple routes and render different components based on the current route.
For example, in a website, React Router can help display different pages depending on the current URL.
React Redux:

It's a library for managing global state in React applications.
It helps maintain and access the application state in a predictable and consistent manner.
For example, if you have data that needs to be accessed by many different components, React Redux can help manage that data effectively.
In summary, while React Router helps manage routes and navigation in your application, React Redux helps manage the global state of your application.

**Reply by fernandabeatriz1** [reply]
Thank you very much.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
